cert_manager/setup.py

```diff
@@ -0,0 +1,64 @@
+# (C) Datadog, Inc. 2019-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from codecs import open  # To use a consistent encoding
```

Comment:
Where is the `pyproject.toml`? Did you happen to migrate an older version of the integration? https://github.com/DataDog/integrations-extras/blob/master/cert_manager/pyproject.toml